README.md

```diff
@@ -1,8 +1,8 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Math Magicians
 
-> Description the project.
+> App created by using REACT that shows you a one page app which include
 
```

Comment:
- [ ] Kindly add your social media links and remove all broken links from your readme, this will make your readme more professional 👍🏿